feat: Pawcut_Tip_Ready UI 그리기

hkwangro opened the pull request:
## 변경 사항 (Changes)

- 835d389 : [Feature] Pawcut_Tip, Ready UI 구현


## 변경 이유 (Reason for Changes)

- Camera 찍기 전 Tip, Ready 화면 구현 했습니다.

## 관련 이슈 (Related Issue)

 - #4 

## 테스트 방법 (How to Test)

-canvas 화면을 첨부했습니다.

<img width="300" height="600" alt="image" src="https://github.com/user-attachments/assets/9be7a48a-4208-4ffd-8ffc-0f7d44c1153d" />

<img width="300" height="600" alt="image" src="https://github.com/user-attachments/assets/9796356e-0474-4787-b44f-2c87ad9864c7" />


## 추가 정보 (Additional Information)
- 없습니다.

delightPIP committed:
Merge pull request #3 from PAWCUT/dev

chore : 프로젝트 폴더구조 세팅

SingleBungIe committed:
feat: 디자인 에셋 추가 #6

luminouxx committed:
Merge pull request #5 from PAWCUT/dev

[Chore] 프로젝트 폴더구조 세팅

luminouxx committed:
Merge pull request #7 from PAWCUT/add-font-color-4

feat: 디자인 폰트&컬러 추가

luminouxx committed:
remove: Font+ 제거

luminouxx committed:
chore: Info.plist에 AppFonts 추가

luminouxx committed:
feat: FontSet 추가

luminouxx committed:
feat: 앱 시작점(@main)에 Font 등록 로직 추가

luminouxx committed:
feat: Font+에 pretendardFont Extension 추가

luminouxx committed:
feat: View+에 커스텀 폰트를 사용할 수 있는 Modifier 추가

luminouxx committed:
remove: Extension에 있던 .gitkeep 제거

luminouxx committed:
Merge pull request #9 from PAWCUT/feat/add-font-set-8

[Feature] Font Set 추가 #8

luminouxx committed:
feat: PawPrimaryButton 컴포넌트 추가

luminouxx committed:
feat: PawPrimaryButton 상하좌우 패딩 조절할 수 있도록 변경

luminouxx committed:
feat: PawSecondaryButton 컴포넌트 추가

luminouxx committed:
feat: PawPrimaryButton에 textPadding 파라미터 추가

luminouxx committed:
feat: PawChoiceButton 컴포넌트 추가

luminouxx committed:
Merge pull request #13 from PAWCUT/feat/add-button-component-10

Feat/add button component 10

luminouxx committed:
feat: PawTextField 컴포넌트 추가

luminouxx committed:
Merge pull request #15 from PAWCUT/feat/add-textfield-component-14

[Feature] TextField Component 제작 #14

hkwangro committed:
feat: Pawcut_Tip_Ready UI 그리기

feat: Pawcut_Tip_Ready UI 그리기

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+import SwiftUI
+
+struct PawcutTip: View {
+    @Environment(\.dismiss) private var dismiss
```
이 부분에 추후에 네비게이션을 동적으로 관리하게 될 것 같은데.
// TODO 주석을 추가해주시는 건 어떨까요?

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+                    }
+                    .padding(.top, 8)
+                    
+                    
```
불필요한 개행은 삭제해주시면 좋을 것 같아요.

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+                            }
+                            .padding(.top, 60)
+                            .padding(.horizontal, 32)
+                            
+                            
```
불필요한 개행은 삭제해주시는 것이 좋을 것 같아요~

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+                            // Tips 리스트
+                            VStack(spacing: 25) {
+                                
+                                TipRow(
+                                    iconName: "tip_camera",
+                                    text: "'촬영하기'를 누르면 바로 촬영이 시작돼요."
+                                )
+                                
+                                
+                                TipRow(
+                                    iconName: "tip_clock",
+                                    text: "6초 타이머가 자동으로 작동해요."
+                                )
+                                
+                                
+                                TipRow(
+                                    iconName: "tip_picture",
+                                    text: "사진은 기본으로 8장이 연속 촬영돼요."
+                                )
+                                
+                                
+                                TipRow(
+                                    iconName: "tip_hand",
+                                    text: "설정에서 카메라 접근을 먼저 허용해 주세요."
+                                )
+                                
+                                
+                                TipRow(
+                                    iconName: "tip_arrow",
+                                    text: "뒤로가기 버튼으로 촬영을 중단할 수 있어요."
+                                )
+                                
```
사소하지만, TopRow가 반복된다면, 해당 데이터를 관리하는 변수 + ForEach를 사용해보는 것이 어떨까 싶네요!

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+                    
+                    // 버튼
+                    VStack(spacing: 0) {
+                        NavigationLink(destination: PawcutReady()) {
```
이 부분도 추후에 수정될 것 같아서 TODO 주석을 추가해주세요!

luminouxx commented on PawCut/PawCut/Sources/Views/PawcutTip.swift:
```diff
@@ -0,0 +1,144 @@
+            
+            Text(text)
+                .pretendardFont(size: ._16, weight: .semibold)
+                .foregroundColor(Color("GrayScale02Color"))
```
`Color(.GrayScale02Color)` 과 같은 형식으로 사용하기로 약속했던 것 같아요!

luminouxx reviewed (changes requested):
광로 고생 많으셨습니다~ 리뷰 사항만 수정해주실 수 있을까요?

hkwangro committed:
feat: Pawcut_Tip_Ready 수정사항 반영

1. // TODO 반영
2. ColorSET 추가 및 칼라 camelCase로 반영
3. TipRow -> 변수 + ForEach 구문으로 변경
4. 기타 불필요한 Padding 삭제

luminouxx reviewed (approved):
고생 많으셨습니다!

luminouxx commented:
광로! base branch만 dev로 바꿔주실 수 있나요!?
